lib/msf/core/modules/external/templates/remote_exploit.erb

```diff
@@ -0,0 +1,43 @@
+require 'msf/core/modules/external/bridge'
+require 'msf/core/module/external'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = <%= meta[:rank] %>
```

Comment:
I think we can pull this into a shared template that looks something like:
```erb
Rank = <%= meta[:rank].capitalize %>Ranking
```

Reply:
Generally only exploits and encoders use rank, but it's available to all modules. Shared template is fine. I like this change!

Reply:
I've decided to move the logic into the shim and keep it for exploits right now. Let me know what you think!